mac-servers/general/cleanup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+rm -rf /Users/administrator/Library/Developer/CoreSimulator/Devices/*
```

Comment:
Won't deleting all the simulator devices cause the unit tests to fail?

Reply:
From what I saw on forums, I understood the folder to be instances of simulator executions, not the underlying files needed to create a simulator instance, which are in `/Library/Developer/CoreSimulator/Profiles/Runtimes`.  As long as we don't want files/apps from previous executions (which we don't) it should be ok.

I also observed the cleanup occur after a reboot of each machine last night and between the overnight schedule and today's first build unit tests have successfully run on all four build servers.

Reply:
Ah ok, thanks! 👍